pypowsybl.dev.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.7-bookworm
```

Comment:
I think it would be better to use a more recent version of python.

Reply:
To build pypowsybl from the sources mentioned that Python >= 3.7 for Linux, Windows and MacOS amd64 is required. That is the reason I put python 3.7

Reply:
Yes but I think in the future python3.7 will be deprecated because some important dependencies does not support python 3.7 any more (for example pandas)